java/lambda-cron/src/main/java/software/amazon/awscdk/examples/LambdaCronStack.java

```diff
@@ -7,12 +7,9 @@
 import software.amazon.awscdk.services.events.RuleProps;
 import software.amazon.awscdk.services.events.Schedule;
 import software.amazon.awscdk.services.events.targets.LambdaFunction;
-import software.amazon.awscdk.services.lambda.Code;
+import software.amazon.awscdk.services.lambda.*;
```

Comment:
I would avoid .* imports for example code. It makes it harder for new developers to know where Classes were imported.